org/cupkee/index.md

```diff
@@ -1,16 +1,9 @@
 ---
-layout: pid
-title: Cupkee
-owner: Cupkee
-license: MIT
-site: http://www.cupkee.cn
-source: https://github.com/cupkee/cupkee
+layout: org
+title: Lixing Ding
```

Comment:
This probably wants to be Cupkee too. Otherwise you probably want to put it in a lixingding org folder instead.